src/authorization-service/auth.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const basicAuthorizer = async (event, context, callback) => {
+  const { type, authorizationToken } = event;
+  if (type !== "TOKEN" || !authorizationToken) {
+    callback("Unauthorized");
+  }
+
+  try {
+    const encodedCreds = authorizationToken.split(" ")[1];
```

Comment:
If I set not only the encoded token to the LocalStorage key (I tried to prepend it with 'Basic '), app doesn't authorizes me. Looks like the deployed version differs somehow from this code